### PR TITLE
#55 Choose Cuisine page

### PR DESCRIPTION
**Pull request by Jen-Lopez**
Addresses Issue #55 

This PR:
- creates Cuisine Carousel
- creates dish modal view
- adds cuisine info in `cuisines.json`

cuisine-overview

dish-view

### CODE REVIEW

**Review by tanyasingh7 — APPROVED** [review]

File: front-end/src/pages/Cuisine.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+.control-dots .dot {
+    background: #000!important;
+}
+
+.carousel-root {
+    margin-top: 50px;
+}
+
+button {
```
Comment (thomastai1666) [inline front-end/src/pages/Cuisine.css:27]: can we move it to app.css or its own button component if it's the default button style? if not, we could also put a classname around it to limit the scope to this page

**Review by thomastai1666 — APPROVED** [review]
the page looks really good! I left one suggestion, but otherwise good to merge

File: front-end/src/pages/Cuisine.css (hunk 1 above)
Comment (Jen-Lopez) [reply front-end/src/pages/Cuisine.css:27]: yup sounds good. I think someone else was working on the button component which is why I didn't add one in but will check again

Comment (Jen-Lopez) [reply front-end/src/pages/Cuisine.css]: once all the frontend is merged I'll use the standard button component but for now I added the class :)

**Pull request merged**